### ISSUE TITLE
Accessibility - ILAB Logo Missing ALT Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#15]
The ILAB logo is missing ALT text so it is reading the name of the file.

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting them to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
Thanks, for adding ALT text for the ILAB logo but the ALT text should be "ILAB" because it is a link to the ILAB website. There is no need to have the word "logo" in the ALT text when it's a link.

You might want to spell out what ILAB means for those that might not know what it is.

If That's the case the ALT text should be "ILAB - The Bureau of International Labor Affairs"

Will keep this issue open until it is fixed.

**Comment by adriennerlong** [comment]
ICF please make ALT text "ILAB - The Bureau of International Labor Affairs"

**Comment by jfc3-dol** [comment]
@adriennerlong,

Closing this issues because the ALT text has been updated to "ILAB - The Bureau of International Labor Affairs".
